src/com/dotmarketing/viewtools/content/ContentMap.java

```diff
@@ -132,7 +132,7 @@ private Object get(String fieldVariableName, Boolean parseVelocity) {
 					}
 				}else if(fieldVariableName.equalsIgnoreCase("title")){
 					ret =  getContentletsTitle();
-				}else if(fieldVariableName.equalsIgnoreCase("structure")){
+				}else if(fieldVariableName.equalsIgnoreCase("structure") || fieldVariableName.equalsIgnoreCase("contenttype")){
```

Comment:
As a rule, we need to do String checks with String literals on the left side to avoid NPEs, e.g.

`"contenttype". equalsIgnoreCase(fieldVariableName)`